java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/model/GenericMqttMessage.java

```diff
@@ -0,0 +1,25 @@
+package com.cumulocity.generic.mqtt.client.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
```

Comment:
suggestion: 'A  Generic MQTT message contains a payload and some associated {@link GenericMqttMetadata metadata}

Reply:
Done.